src/EventStore.Transport.Http/Codecs/JsonCodec.cs

```diff
@@ -16,7 +16,7 @@ public class JsonCodec : ICodec {
 			ContractResolver = new CamelCasePropertyNamesContractResolver(),
 			DateParseHandling = DateParseHandling.None,
 			NullValueHandling = NullValueHandling.Ignore,
-			DefaultValueHandling = DefaultValueHandling.Ignore,
+			DefaultValueHandling = DefaultValueHandling.Include,
```

Comment:
This change will alter the behaviour for pretty much anything that uses json over http.. are we sure this is a good idea @pgermishuys @hayley-jean ?

Reply:
Judging by where this is used, this should be fine and is likely what we actually want to happen in the first place